tests/legacy/test_stream_counter_readers_writers.py

```diff
@@ -111,25 +114,26 @@ def test_one_sample_double(self, real_x_series_device, seed):
         # Select random counters from the device.
         counters = random.sample(self._get_device_counters(real_x_series_device), 2)
 
-        with nidaqmx.Task() as write_task, nidaqmx.Task() as read_task:
-            write_task.co_channels.add_co_pulse_chan_freq(counters[0], freq=frequency)
-            write_task.timing.cfg_implicit_timing(sample_mode=AcquisitionType.CONTINUOUS)
-            actual_frequency = write_task.co_channels.all.co_pulse_freq
+        write_task = generate_task()
+        read_task = write_task()
```

Comment:
The 2nd line should call `generate_task()`:
```
=============================================== short test summary info ===============================================
_kwargs-5024615685632901870] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_multi_sample_double[library_init_kwargs-7404219599611537002] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_one_sample_pulse_freq[library_init_kwargs-7384356521189664592] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_many_sample_pulse_freq[library_init_kwargs-8740148906740844940] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_one_sample_pulse_time[library_init_kwargs-2634622406642771584] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_many_sample_pulse_time[library_init_kwargs-1050633190093921164] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_pulse_ticks_1_samp[library_init_kwargs-1344567384821241741] - TypeError: 'Task' object is not callable
FAILED tests/legacy/test_stream_counter_readers_writers.py::TestCounterReaderWriter::test_many_sample_pulse_ticks[library_init_kwargs-8857678428894285084] - TypeError: 'Task' object is not callable
=================================== 8 failed, 210 passed, 345 deselected in 14.85s ====================================
```

Reply:
fixed

Reply:
There are more copies of this bug.

```
PS D:\dev\nidaqmx-python> rg "read_task = write_task" tests
tests\legacy\test_stream_counter_readers_writers.py
148:        read_task = write_task()
186:        read_task = write_task()
218:        read_task = write_task()
271:        read_task = write_task()
302:        read_task = write_task()
356:        read_task = write_task()
396:        read_task = write_task()
```

Reply:
I have fixed all such occurences